util/chplenv/chpl_aux_filesys.py

```diff
@@ -20,6 +21,24 @@ def get():
             stderr.write("Warning: Can't find your Java installation\n")
         if not found_hdfs:
             stderr.write("Warning: Can't find your Hadoop installation\n")
+
+    elif aux_fs == 'hdfs3':
+        def fetchInfo(env, val, err):
+            stripstr = (env.find('INCLUDE') > 0 and '-I' or '-L')
```

Comment:
Why should fetchInfo look for INCLUDE in the environment variable argument you give it? Wouldn't it be smarter just to pass -I or -L or another argument?